### ISSUE TITLE
Crash on commenting SQL if dbeaver use external formatting

### ISSUE THREAD

**Issue by ay4t** [dbeaver/dbeaver#22140]
### Description

If I use external formatting in preferences and then press the (-) button, DBeaver immediately crashes and becomes not responding.

OS: Windows 10 Pro x64
Version: 22H2
OS Build: 19045
DBeaver version:  23.3.0.202312031432

https://github.com/dbeaver/dbeaver/assets/87152520/2f30a946-a088-4d49-a0b5-515f9e71db82

### DBeaver Version

23.3.0.202312031432

### Operating System

Windows 10 Pro x64

### Database and driver

1. Database name and version: 10.11.4-MariaDB
2. Driver name and version: MariaDB JDBC Driver

### Steps to reproduce

1. change preference to external formatting like sqlinform e.g: "C:\sqlinform\cli.exe" -dbeaver ${file}
2. and write simple query
3. push ( - ) button, 

i want to comment shome condition but dbeaver always crash. if i turn back to default formatter it will fine.

**Comment by LonwoLonwo** [comment]
Hello @ay4t 

Could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Comment by Destrolaric** [comment]
Hi,
Could you also please try to generate a thread dump and attach it here? Here is the instruction: https://dbeaver.com/docs/wiki/Making-a-thread-dump/
Thank you.

**Comment by ay4t** [comment]
Of course. I'll attach the log soon. Thank you for responding to this issue

**Comment by ay4t** [comment]
Of course. I'll attach the log soon. Thank you for responding to this issue

**Comment by ay4t** [comment]
> Hello @ay4t 
> 
> Could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

Thank you for responding. I will follow the instructions and send the attachment.

**Comment by ay4t** [comment]
> Hi,
> Could you also please try to generate a thread dump and attach it here? Here is the instruction: https://dbeaver.com/docs/wiki/Making-a-thread-dump/
> Thank you.
> 

Of course. I'll attach the log soon. Thank you for responding to this issue

**Comment by ay4t** [comment]
> Hi, Could you also please try to generate a thread dump and attach it here? Here is the instruction: https://dbeaver.com/docs/wiki/Making-a-thread-dump/ Thank you.

Hi. 
After I tried installing JDK version 21, I was planning to run the 'jstack' command for a thread dump. However, when I tried again, the error didn't appear anymore.

Initially, before I installed JDK, I was using the "Read database table/columns for semantic analysis" setting, and every time I pressed the (-) button to comment out syntax, the program immediately became unresponsive.

![dbeaver_crfLTyvQZ0](https://github.com/dbeaver/dbeaver/assets/87152520/bd1f0b20-9b1f-4d05-8c5b-6f8ce3e7eec9)
